parsec/core/gui/tr/parsec_en.po

```diff
@@ -2752,3 +2731,45 @@ msgstr "Create a new workspace"
 msgid "ACTION_WORKSPACE_GOTO_FILE_LINK"
 msgstr "Go to a file using a link"
 
+#~ msgid "TEXT_FILE_INFO_LOCATION"
```

Comment:
Is it normal that you add comments ?

Reply:
Good catch!
It has been fixed, but here's the why and how it happened.

Translations files are a bit weird. They're generated automatically from .py and .ui files using `python setup.py extract_translations`. It takes the files that are listed in the .pro (Qt project file) and extract all the translation strings into the .po files.
Sadly, a `\` was missing into the .po file (that's how you know it's a good format: when you need to separate your file list by backslashes) meaning files_info_widget.ui was not taken into account and it automatically commented the strings in the .po files.